### PR TITLE
Archi: Update to 4.5.1 & remove 32bit

### PR DESCRIPTION
**Pull request by hoilc**
https://www.archimatetool.com/downloads/What's%20New%20in%20Archi.pdf

Starting from 4.5, Archi supports only 64-bit versions.

#2308

### CODE REVIEW

File: bucket/archi.json [hunk 1]
```diff
@@ -2,41 +2,35 @@
...
+            "archi.exe",
+            "Archi"
+        ]
+    ],
     "checkver": {
         "url": "https://www.archimatetool.com/download/",
         "re": "data-version=\"([\\d.]+)\""
     },
-    "extract_dir": "Archi",
+
```
Comment (Ash258) [inline bucket/archi.json]: Empty line
```suggestion
```

File: bucket/archi.json [hunk 2]
```diff
@@ -2,41 +2,35 @@
     "homepage": "https://www.archimatetool.com/",
     "license": "MIT",
     "description": "Archi® is a free, open source, cross-platform tool and editor to create ArchiMate models.",
```
Comment (Ash258) [inline bucket/archi.json]:
```suggestion
    "description": "Free, open source, cross-platform tool and editor to create ArchiMate models.",
```

**Comment by hoilc** [comment]
> Move `extract_dir`, `bin` , `shortcuts` properties after `architecture`. (preserve order which I specified)

I was also suggested to put `homepage` and `description` at the top in another PR.

Maybe a tool that adjusts the field order automatically is needed.😂

**Comment by Ash258** [comment]
It would be nice, but it is hard to create such. Automatic deterministic properties sorting needs to cover lots of combinations especially nested sorting of architecture property.

**Pull request merged**